pkg/kubelet/kubelet.go

```diff
@@ -1361,6 +1361,23 @@ func (kl *Kubelet) syncLoop(updates <-chan PodUpdate, handler SyncHandler) {
 	}
 }
 
+// Returns Docker version for this Kubelet.
+func (kl *Kubelet) GetDockerVersion() (string, error) {
+	if kl.dockerClient == nil {
+		return "", fmt.Errorf("No Docker client")
```

Comment:
You can ask me why :)

It's because error strings are typically concatenated together as the error rolls up the call stack and generally printed following some enclosing context, e.g.:

if err != nil {
  return fmt.Errorf("error here: %v", err)
}

or

if err != nil {
  glog.Error("Error doing X: %v")
}

This keeps spurious capital letters out of the middle of the error strings, making them easier to read.